airflow/providers/snowflake/transfers/s3_to_snowflake.py

```diff
@@ -85,6 +86,15 @@ def __init__(
         session_parameters: Optional[dict] = None,
         **kwargs,
     ) -> None:
+        warnings.warn(
+            """
+            SnowflakeToSlackOperator is deprecated.
```

Comment:
This is wrong class name.
Raised https://github.com/apache/airflow/pull/26047 to fix it